test/test_simulator_server.py

```diff
@@ -359,3 +359,42 @@ def test_bad_set_filename_command(self):
         self.client.send_message(request)
         reply = self.client.read_message()
         self.assertEqual(reply.code, grammar.FAIL)
+
+    def test_set_enable_command(self):
+        request = grammar.Message(message_type = grammar.REQUEST,
+                                  name = "set-enable",
+                                  arguments = ['0', '1'])
+        self.client.send_message(request)
+        reply = self.client.read_message()
+        self.assertEqual(reply.code, grammar.OK)
+
+    def test_set_enable_less_than_2_feeds(self):
+        request = grammar.Message(message_type = grammar.REQUEST,
+                                  name = "set-enable")
+        self.client.send_message(request)
+        reply = self.client.read_message()
+        self.assertEqual(reply.code, grammar.FAIL)
+
+    def test_set_enable_wrong_parameter_type(self):
+        request = grammar.Message(message_type = grammar.REQUEST,
```

Comment:
One recomendetion from [PEP8](https://peps.python.org/pep-0008/#other-recommendations) is to not use spaces around the `=` sign when used to indicate a keyword argument. As usual, if you put spaces consistently around `=` in all keyword arguments of the codebase, you can keep this code as it is

Reply:
I used the same style it was already in use in the file